### ISSUE TITLE
Animation bug after serialize

### ISSUE THREAD

**Issue by dad72** [BabylonJS/Babylon.js#1835]
Hello,

I found one big bug about animation when one mesh is Serializing

On this PG I reproduce the problem, I create a character without serialization, then I serializes and I load this new mesh and becomes distorted if I animates:

http://www.babylonjs-playground.com/#GA8XI#64

I feel that the Skeleton is not resize when you rescale the mesh after serialization.

**Comment by deltakosh** [comment]
Hello, you can fix it by serializing the mesh BEFORE setting the scaling:
http://www.babylonjs-playground.com/#GA8XI#65

The problem comes from the skeleton which is defined to use the pose matrix (the initial matrix) from the mesh. It does not come with it. So you cannot change the  mesh's world matrix before the pose matrix is built (it happens at loading time)

**Comment by dad72** [comment]
The problem is that I can not do like this. The scaling is defined before I serialize by the user. I can not define the scale after serialization.

The user puts to scale to import the mesh so that it can add to the scene to scale. Can I save this mesh in a file with this scale, position, rotation ... but when I reload the file, to recharge the stage, this problem is present.

The problem is that all the models create with 3ds max, are always big. For example, if I made a 1.80m model, in Babylon he is 180. So I always rescale in Babylon. But if I modeled in 3ds max with too small a scale of 1.80m, there are accuracy problems the zoom and other because it is too small. I buy many models long ago and they all big. Even for download free models, they are big.

Thanks DK
